pkg/operator/resources/apisplitter/status.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2020 Cortex Labs, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package apisplitter
+
+import (
+	"sort"
+	"time"
+
+	"github.com/cortexlabs/cortex/pkg/lib/errors"
+	"github.com/cortexlabs/cortex/pkg/operator/config"
+	"github.com/cortexlabs/cortex/pkg/operator/operator"
+	"github.com/cortexlabs/cortex/pkg/types/status"
+	istioclientnetworking "istio.io/client-go/pkg/apis/networking/v1alpha3"
+)
+
+const _stalledPodTimeout = 10 * time.Minute
+
+func GetStatus(apiName string) (*status.Status, error) {
+	var virtualService *istioclientnetworking.VirtualService
+
+	virtualService, err := config.K8s.GetVirtualService(operator.K8sName(apiName))
+	if err != nil {
+		return nil, err
+	}
+	if virtualService == nil {
+		return nil, errors.ErrorUnexpected("unable to find trafficsplitter", apiName)
```

Comment:
Some parts of the code like this one refer to the APISplitter as trafficsplitter. We should make sure that we refer to APISplitter consistently to not confuse the users and ourselves when we are debugging errors